manual/source/Tutorial/GettingStarted/Metafunctions.rst

```diff
@@ -97,10 +120,13 @@ For example the following function prints the length of a fixed sized string usi
 .. includefrags:: demos/tutorial/metafunctions/base.cpp
     :fragment: length
 
-SeqAn Metafunctions
--------------------
+.. includefrags:: demos/tutorial/metafunctions/base.cpp.stdout
+    :fragment: length
+
+.. important::
+      Redundant use of "*Value*":
 
-If you want to search for metafunctions only you can do so by only selecting the metafunction category to the left of the search window at the online documentation.
+      There is a **Type Metafunction** called :dox:`ContainerConcept#Value` (``Value<TSomeType>::Type``), which has nothing to do with the **Value Metafunctions** described above.
 
```

Comment:
A little "colloquial". Maybe:
> Please note that :dox:`ContainerConcept#Value` (``Value<TSomeType>::Type``) is a **Type Metafunction**, because it returns a Type (e.g. of Values in a container) and not a value.